app-i18n/enca/enca-1.19.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils flag-o-matic multilib-minimal toolchain-funcs
+
+DESCRIPTION="Detect and convert encoding of text files"
+HOMEPAGE="https://cihar.com/software/enca/"
+SRC_URI="https://dl.cihar.com/${PN}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~x86-freebsd ~amd64-linux ~ia64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
+IUSE="doc +iconv recode"
+
+RDEPEND="
+	iconv? ( virtual/libiconv[${MULTILIB_USEDEP}] )
+	recode? ( app-text/recode[${MULTILIB_USEDEP}] )
+"
+DEPEND="${RDEPEND}
+	sys-devel/gettext
+	doc? ( dev-util/gtk-doc )
+"
+
+pkg_pretend() {
+	if tc-is-cross-compiler && use iconv; then
+		die "${PN} can't be cross built with iconv USE enabled. See Gentoo bug 593220."
+	fi
+}
+
+src_prepare() {
+	default_src_prepare
```

Comment:
Sorry, it just seems that ``default`` is much more commonly used here. github finds only 33 occurrences of ``default_src_prepare`` in the tree.